pkg/util/file.go

```diff
@@ -13,3 +17,22 @@ func SetFileModeForPath(name string, mode os.FileMode) error {
 func SetFileModeForFile(file *os.File, mode os.FileMode) error {
 	return file.Chmod(mode)
 }
+
+// ReadFile reads from a file
+func ReadFile(path string) (string, error) {
+	if path == "" {
+		return "", nil
+	}
+
+	for {
```

Comment:
I would suggest adding a failure state is after 5min or something, the path still does not exist.

Reply:
yeah, good point, let me replace the infinite loop with something else using time. I copied/pasted this function from another place though, and it never happened before, but better safe than sorry